src/App.tsx

```diff
@@ -1,10 +1,81 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { ErrorMessage } from './components/ErrorMessage/ErrorMessage';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+
+const USER_ID = 10897;
+
+const testTodos: Todo[] = [
+  {
+    id: 1,
+    userId: USER_ID,
+    title: 'Hi',
+    completed: true,
+  },
+  {
+    id: 2,
+    userId: USER_ID,
+    title: 'Hi',
+    completed: false,
+  },
+  {
+    id: 3,
+    userId: USER_ID,
+    title: 'Hi',
+    completed: false,
+  },
+  {
+    id: 4,
+    userId: USER_ID,
+    title: 'Hi',
+    completed: false,
+  },
+];
```

Comment:
What's the point if you're loading todos from API?

Reply:
@silchyn mate-api was not avaliable on this weekend, so we had to create a few todos to work on this task.